G/Givaro/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Givaro"
+version = v"4.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/linbox-team/givaro/releases/download/v$version/givaro-$version.tar.gz", "865e228812feca971dfb6e776a7bc7ac959cf63ebd52b4f05492730a46e1f189"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/givaro-4.2.0
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p0 ${f}
+done
+
+autoreconf
+./configure CCNAM=${CC} CPLUS_INCLUDE_PATH=$includedir --prefix=$prefix --build=${MACHTYPE} --host=${target}
+
+# really ugly! but I see no other solution for now.
+# this patch modifies libtool so that it doesn't use the not-yet-implemented "-r" flag for the linker, replacing it by -dynamiclib
+if [[ ${target} =~ aarch64-apple-darwin* ]]; then
+    echo "Patching libtool so it doesn't use ld -r"
+    patch -p0 < ${WORKSPACE}/srcdir/patches/libtool-aarch64-apple-darwin.hack
+fi
+
```

Comment:
Isn't it easier not to use `lld` with `clang`?  Which means, add the option `clang_use_lld=false` to `build_tarballs` below.